news/_posts/2022-10-30-playtest-20221030.md

```diff
@@ -0,0 +1,87 @@
+---
+kind: article
+title: "Playtest 20221030"
+author: "PunkPun"
+created_at: 2022-10-30 22:00 +0000
+disqus_id: "playtest-20221030"
+permalink: "/news/playtest-20221030/"
+---
+
+Welcome back, Commanders! After more than a year and a half the OpenRA team proudly presents [Playtest 20221030]({{ '/download' | relative_url }}). It is finally here!
+
+There have been a ton of refactors under the hood -- it took us a very long time to catch and squash all those pesky bugs -- and we are excited to finally showcase a number of new features and improvements that have been built up over the last year and a half.
+
+One highlight feature is the new hierarchical pathfinder, which massively increases performance on large or more complex maps.
+
+<figure>
+  <img src="{{ '/images/news/20221031-hpf.png' | relative_url }}" style="width: 600px" alt="HPF helps units better predict obstacles ahead of them" />
+  <figcaption>HPF helps units to better predict obstacles ahead of them.</figcaption>
+</figure>
+
+Our networking code has had some major improvements in both low and high latency games.
```

Comment:
Maybe
```suggestion
Our new netcode brings major improvements in both low and high latency games. Clients that experience connection hiccups can now catch up and do not slow the game permanently.
```
if you want more than one sentence. (It still irks me a bit that this reads like the netcode has improvements during games... while it is the connection that sees those improvements.)

Reply:
it's not just the connection though. The way we handle the delay is different

Reply:
it's not just about the hiccups, the whole game just plainly runs smoother and is more responsive

Reply:
Then mention this too. How is an end-user supposed to know this from the single sentence?

Reply:
Sure